docs/gen2_Tutorial.md

```diff
@@ -429,6 +429,73 @@ and clean them up.
+- run `arlon cluster create` with the repo-path pointing to the basecluster manifest described in setp 3, set the profile to be the one created in step 2 and pass the `autoscaler` flag
+
+### Bundle creation
+
+Register a dynamic bundle pointing to the bundles/capi-cluster-autoscaler in the Arlon repo.
+
+The capi-cluster-autoscaler bundle requires the name of the cluster, so that it knows what namespace in the management cluster to scan for CAPI resources. To enable the cluster-autoscaler bundle, add one more parameter during cluster creation: `srcType`. This is the ArgoCD-defined application source type (Helm, Kustomize, Directory).
+
+This example creates a bundle pointing to the bundles/capi-cluster-autoscaler in Arlon repo
+
+```shell
+arlon bundle create cas-bundle --tags cas,devel,test --desc "CAS Bundle" --repo-url https://github.com/arlonproj/arlon.git --repo-path bundles/capi-cluster-autoscaler --srctype helm
+```
+
+### Profile creation
+
+Create a profile that contains this capi-cluster-autoscaler bundle.
+
+```shell
+arlon profile create dynamic-cas --repo-url ${WORKSPACE_REPO_URL} --repo-base-path profiles --bundles cas-bundle --desc "dynamic cas profile" --tags examples
+```
+
+### manifest directory preparation
+
+Two additional properties `cas-max` and `cas-min` are used to set 2 annotations for Max/Min nodes on MachineDeployment required by the cluster autoscaler for CAPI as a part of the manifest directory preparation. These are the annotations required by the MachineDeployment for autoscaling. 
+
+Note: These are the default values for the `cas-min` and `cas-max` properties
+```shell
+ annotations:
+     cluster.x-k8s.io/cluster-api-autoscaler-node-group-min-size: '1'
+     cluster.x-k8s.io/cluster-api-autoscaler-node-group-max-size: '9'
+```
+
+These annotations are added during the preparegit step. If these annotations are already present in the manifest file, then they will not be added again as a part of `preparegit`
+
+Preparing the git directory to serve as the bascluster: 
+
+```shell
+arlon basecluster preparegit --repo-path <pathToDirectory> --cas-min 1 --cas-max 9 --repo-url <repoUrl> 
+```
+
+### manifest directory validation
+
+To determine if a git directory is eligible to serve as base cluster, run the `basecluster validategit` command:
+
+```shell
+arlon basecluster preparegit --repo-path <pathToDirectory> --repo-url <repoUrl> 
+```
+
+### gen2 cluster creation with autoscaling enabled
+
+To add CAS support for gen2 clusters, the cluster create sub-command of the arlon CLI has a `autoscaler` flag which deploys the capi-cluster-autoscaler helm chart on the management cluster.
+
+To create a gen2 workload with a cluster-autoscaler pod running, from the base cluster, run this command:
+
+Note: Use the dynamic profile that was created in the previous steps
+
+```shell
+arlon cluster create --cluster-name <clusterName> --repo-url <repoUrl> --repo-path <pathToDirectory> --profile dynamic-cas --autoscaler
```

Comment:
```suggestion
arlon cluster create --cluster-name <clusterName> --repo-url ${WORKSPACE_REPO} --repo-path <pathToDirectory> --profile dynamic-cas --autoscaler
```

Reply:
Should I change it all the places in this documentation or let it be like this to be consistent with the entire doc ?

Reply:
Keeping it consistent with rest of the docs